[2023-11-06T12:06:03.851+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-21T00:00:00+00:00 [queued]>
[2023-11-06T12:06:03.868+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-21T00:00:00+00:00 [queued]>
[2023-11-06T12:06:03.869+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:06:03.993+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-21 00:00:00+00:00
[2023-11-06T12:06:04.001+0000] {standard_task_runner.py:57} INFO - Started process 856 to run task
[2023-11-06T12:06:04.012+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-21T00:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpp0f48kzc']
[2023-11-06T12:06:04.019+0000] {standard_task_runner.py:85} INFO - Job 95: Subtask task_a
[2023-11-06T12:06:04.149+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-21T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:06:04.321+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-21T00:00:00+00:00'
[2023-11-06T12:06:04.326+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:06:04.328+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World :)']
[2023-11-06T12:06:04.351+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:06:04.357+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `)'
[2023-11-06T12:06:04.358+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo Hello World :)'
[2023-11-06T12:06:04.359+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-06T12:06:04.381+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-06T12:06:04.389+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231021T000000, start_date=20231106T120603, end_date=20231106T120604
[2023-11-06T12:06:04.507+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 95 for task task_a (Bash command failed. The command returned a non-zero exit code 2.; 856)
[2023-11-06T12:06:04.556+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-06T12:06:04.596+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-06T12:19:07.058+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-21T00:00:00+00:00 [queued]>
[2023-11-06T12:19:07.089+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-21T00:00:00+00:00 [queued]>
[2023-11-06T12:19:07.090+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:19:07.142+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-21 00:00:00+00:00
[2023-11-06T12:19:07.152+0000] {standard_task_runner.py:57} INFO - Started process 2057 to run task
[2023-11-06T12:19:07.159+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-21T00:00:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpnh0l6d1r']
[2023-11-06T12:19:07.172+0000] {standard_task_runner.py:85} INFO - Job 167: Subtask task_a
[2023-11-06T12:19:07.285+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-21T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:19:07.469+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-21T00:00:00+00:00'
[2023-11-06T12:19:07.471+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:19:07.472+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World.']
[2023-11-06T12:19:07.488+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:19:07.492+0000] {subprocess.py:93} INFO - Hello World.
[2023-11-06T12:19:07.493+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-06T12:19:07.530+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231021T000000, start_date=20231106T121907, end_date=20231106T121907
[2023-11-06T12:19:07.580+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-06T12:19:07.604+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
